pkg/server/container_create.go

```diff
@@ -438,9 +438,11 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount) {
 	for _, mount := range mounts {
 		dst := mount.GetContainerPath()
 		src := mount.GetHostPath()
-		options := []string{"rw"}
+		options := []string{"rbind", "rprivate"}
```

Comment:
I feel like these options, rbind (recursive) and rprivate (isolated), should be be derived through the CRI create request, possibly also the pod. Is this the current default for dockershim? If so it makes sense to make it the default if there is no override from the kublet through CRI. Thoughts?